pages/learn/deploy/offline-updates.md

```diff
@@ -72,25 +79,25 @@ $ ssh-keygen -o -a 100 -t ed25519 -f id_ed25519 -C 'offline-updates' -N ''
 $ ssh_key=id_ed25519.pub
 ```
 
-### Create/Use Pre-existing Application
+### Create/Use Pre-existing fleet's
 
 Initialize the `arch` and `device-type` environment variable with correct CPU architecture (`aarch64`, `armv7l`, etc.) and device type (`raspberrypi4-64`, `raspberrypi3`, etc.). The list of names for supported device types and their architectures can be found on the [hardware][supported-devices-list] page.
 
-If a pre-existing application needs to be re-used, then initialize the `app_name` variable with that application's name.
+If a pre-existing fleet's needs to be re-used, then initialize the `app_name` variable with that fleet's's name.
 
 ```bash
 $ arch=<CPU ARCHITECTURE TYPE>
 $ device_type=<DEVICE_TYPE>
 $ app_name=offline-${arch}
 ```
 
-These environment variables will be used later in the process. If a pre-existing application is needed to be used, then the next step can be skipped. Otherwise, create a new balenaCloud application by running [`balena app create`](https://www.balena.io/docs/reference/balena-cli/#app-create-name).
+These environment variables will be used later in the process. If a pre-existing fleet's is needed to be used, then the next step can be skipped. Otherwise, create a new balenaCloud fleet's by running [`balena app create`](https://www.balena.io/docs/reference/balena-cli/#app-create-name).
```

Comment:
```suggestion
These environment variables will be used later in the process. If a pre-existing fleet needs to be used, then the next step can be skipped. Otherwise, create a new balenaCloud fleet by running [`balena fleet create`](https://www.balena.io/docs/reference/balena-cli/#fleet-create-name).
```

^ unsure if anchor is correct